remote-api/src/main/java/org/alfresco/rest/workflow/api/impl/MapBasedQueryWalker.java

```diff
@@ -247,25 +248,31 @@ else if (type == WhereClauseParser.LESSTHANOREQUALS)
             }
             else
             {
-                throwError = !handleUnmatchedComparison(type, propertyName, propertyValue);
+                throwError = !handleUnmatchedComparison(type, propertyName, propertyValue);
             }
         }
         else
         {
             throwError = !handleUnmatchedComparison(type, propertyName, propertyValue);
         }
 
-        if (throwError) 
-        { 
-            throw new InvalidArgumentException("framework.exception.InvalidProperty", new Object[] {propertyName, propertyValue, WhereClauseParser.tokenNames[type]});
-        }
-        else if (negated)
-        {
-            // Throw error for the unsupported negation only if the property was valid for comparison, show the more meaningful error first.
-            throw new InvalidArgumentException("Cannot use NOT for " + WhereClauseParser.tokenNames[type] + " comparison."); 
+        if (throwError)
+        {
+            throw new InvalidArgumentException("framework.exception.InvalidProperty", new Object[] {propertyName, propertyValue, WhereClauseParser.tokenNames[type]});
+        }
+        else if (negated)
+        {
+            // Throw error for the unsupported negation only if the property was valid for comparison, show the more meaningful error first.
+            throw new InvalidArgumentException("Cannot use NOT for " + WhereClauseParser.tokenNames[type] + " comparison.");
         }
     }
 
+    @Override
+    public Collection<String> getProperty(String propertyName, int type, boolean negated)

```

Comment:
This my be hard to notice, but adding this method should be the only change in this class.